code/datums/diseases/japanesefever.dm

```diff
@@ -0,0 +1,92 @@
+/datum/disease/japanesefever
+	name = "Japanese Fever"
+	desc = "Oh. Oh no. Oh no no no."
```

Comment:
Might want to give more of a medical description

Reply:
Doesn't actually do anything, I believe

Reply:
It shows up on health analyzers IIRC

Reply:
Oh, alright. I'll change it then